From c8088b3689c37ea0c60df4423d0fa4a390320d62 Mon Sep 17 00:00:00 2001
From: Sun Yijin <yijin.sun@intel.com>
Date: Tue, 10 Jul 2012 17:31:59 +0800
Subject: Generate gzip package of system.img

BZ: 46708

[BOOT CAMP SPECIFIC]

Domian: AOSP-Build

Change-Id: Ib1edc6bb6bfa1df2bb413d001c2bc1d5535fc20c
Orig-Change-Id: I461fba34f13e6717dd17a2cfb2576ae5632d391c
Category: device enablement
Domain: AOSP-build
Origin: internal
Upstream-Candidate: no
Reviewed-on: http://android.intel.com:8080/74479
---
 core/Makefile |   12 ++++++++++++
 1 files changed, 12 insertions(+), 0 deletions(-)

diff --git a/core/Makefile b/core/Makefile
index 4e01279..305113b 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -933,6 +933,18 @@ $(warning Warning: with dexpreopt enabled, you may need a full rebuild.)
 endif
 endif
 
+# -----------------------------------------------------------------
+# Zip package of system.img
+#
+INSTALLED_SYSTEMIMG_GZ_TARGET := $(PRODUCT_OUT)/system.img.gz
+
+$(INSTALLED_SYSTEMIMG_GZ_TARGET) : $(INSTALLED_SYSTEMIMAGE) | $(MINIGZIP)
+	@echo "Generate system.img.gz"
+	$(MINIGZIP) <$? >$@
+
+.PHONY: systemimg_gz
+systemimg_gz: $(INSTALLED_SYSTEMIMG_GZ_TARGET)
+
 #######
 ## system tarball
 define build-systemtarball-target
-- 
1.7.4.1

